.github/workflows/distribute-grain.yml

```diff
@@ -0,0 +1,63 @@
+name: Distribute Grain
+on:
+  # A new Cred interval every Sunday. We'll distribute Grain 5 minutes
+  # after the new interval
+  schedule:
+    - cron: 5 0 * * 0
+
+jobs:
+  GenerateCredInstance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # Required to make github pages deployment work correctly
+
+      # We load the cache data so we can explicitly clear and re-generate it
+      # This ensures the cache is purged weekly, and ensures that future runs will use the same
```

Comment:
This is confusing; I certainly didn’t understand the purpose on initial
read. From conversations, it sounds like the purpose of this is to incur
a side-effect of clearing the cache weekly, unrelated to the grain
distribution.

Why don’t we instead simply remove all caching behavior from this job
(so that it starts with a clean cache), then have a separate weekly cron
to clear the cache if we want that behavior? Then this job is much more
clearly correct, and it doesn’t conflate concerns.

Reply:
Removed caching behavior; can add clearing later. This new workflow
clearly does not use the cache because it never loads it.